packages/iov-bns/src/types.ts

```diff
@@ -188,6 +271,25 @@ export function isUpdateEscrowPartiesTx(tx: LightTransaction): tx is UpdateEscro
   return tx.kind === "bns/update_escrow_parties";
 }
 
+// Transactions: Governance
+
+export interface CreateProposalTx extends LightTransaction {
+  readonly kind: "bns/create_proposal";
+  readonly title: string;
+  /** The transaction to be executed when the proposal is accepted */
+  readonly option: ProposalOption;
+  readonly description: string;
+  readonly electionRuleId: Uint8Array;
+  /** Unix timestamp when the proposal starts */
+  readonly startTime: number;
+  /** The author of the proposal must be included in the list of transaction signers. */
+  readonly author: Address;
+}
```

Comment:
The electorate is inferred from the election rule? Is the ID of an election rule guaranteed to be unique?

Reply:
1. Yes
2. Yes, the `electionRuleId` must be pulled from the blockchain and is guaranteed to be unique there